<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QGroupBox" name="groupBox_input">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_input">
      <item row="0" column="0">
       <widget class="QLabel" name="label_video_path">
        <property name="text">
         <string>Video Path</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_video_path"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_fileExplorer_input">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_trim">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Trimming</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_trim">
      <item row="0" column="1">
       <widget class="QLabel" name="label_start_trim">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_start_trim"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_end_trim"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_end_trim">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_output">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_output">
      <item row="3" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_output_directory"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox_frame_rate">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_taget_frame_rate">
        <property name="text">
         <string>Target Frame Rate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QCheckBox" name="checkBox_image_plane">
        <property name="text">
         <string>Create Image Plane</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_output_directory">
        <property name="text">
         <string>Output Directory</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_fileExplorer_output">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_sequence_file_name">
        <property name="text">
         <string>Sequence File Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="comboBox_file_extension">
        <item>
         <property name="text">
          <string>.jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.png</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_file_extension">
        <property name="text">
         <string>File Extension</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_sequence_file_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_create_image_sequence">
     <property name="text">
      <string>Create Image Sequence</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
